app/src/models/HighImpactActionRanked.ts

```diff
@@ -0,0 +1,207 @@
+import * as Sequelize from "sequelize";
+import { DataTypes, Model, Optional } from "sequelize";
+
+export interface HighImpactActionRankedAttributes {
+  id: string;
+  hiaRankingId: string;
+  type: string;
+  name: string;
+  hazard?: string[];
+  sector?: string[];
+  subsector?: string[];
+  primaryPurpose?: string[];
+  description?: string;
+  dependencies?: string[];
+  cobenefits?: Record<string, any>;
+  equityAndInclusionConsiderations?: string;
+  GHGReductionPotential?: Record<string, any>;
+  adaptationEffectiveness?: string;
+  adaptationEffectivenessPerHazard?: Record<string, any>;
+  costInvestmentNeeded?: string;
+  timelineForImplementation?: string;
+  keyPerformanceIndicators?: string[];
+  powersAndMandates?: string[];
+  biome?: string;
+  isSelected?: boolean;
+  actionId: string;
+  rank: number;
+  explanation: { en: string; es: string; pt: string };
+  lang: string;
+  created?: Date;
+  lastUpdated?: Date;
+}
+
+export type HighImpactActionRankedPk = "id";
+export type HighImpactActionRankedId = HighImpactActionRanked[HighImpactActionRankedPk];
+export type HighImpactActionRankedCreationAttributes = Optional<HighImpactActionRankedAttributes, "id">;
+
+export class HighImpactActionRanked
+  extends Model<HighImpactActionRankedAttributes, HighImpactActionRankedCreationAttributes>
+  implements HighImpactActionRankedAttributes
+{
+  id!: string;
+  hiaRankingId!: string;
+  actionId!: string;
+  rank!: number;
+  explanation!: { en: string; es: string; pt: string };
```

Comment:
This isn't going to work (we will keep adding languages). Let's use `Record<string, string>` here instead (if you want you can check the key to be one of the values of `LANGUAGES` from `types.ts`, or even better, `languages` from `i18n/settings.ts`). And let's make sure we aren't hard-coding these possible languages into the database structure.